cms/service/ScoringService.py

```diff
@@ -100,6 +100,9 @@ def execute(self, entry):
                 submission_result.ranking_score_details = \
                 score_type.compute_score(submission_result)
 
+            if submission_result.scored_at is None:
+                submission_result.scored_at = make_datetime()
```

Comment:
Can we add test that checks that it is not None after scoring, somewhere around here: https://github.com/ioi-2022/cms/blob/master/cmstestsuite/unit_tests/service/ScoringServiceTest.py#L100?